www-apps/gitea/gitea-1.13.0.ebuild

```diff
@@ -33,14 +33,11 @@ RDEPEND="${COMMON_DEPEND}
 	dev-vcs/git"
 
 DOCS=(
-	custom/conf/app.ini.sample CONTRIBUTING.md README.md
+	custom/conf/app.example.ini CONTRIBUTING.md README.md
 )
 FILECAPS=(
 	cap_net_bind_service+ep usr/bin/gitea
 )
-PATCHES=(
-	"${FILESDIR}/1.12-fix-vendoring.patch"
-)
 
 RESTRICT="test"
```

Comment:
I'd still really love even a small number of tests to be running in future if possible